cMake/FreeCAD_Helpers/FindSalomeMESH.cmake

```diff
@@ -0,0 +1,126 @@
+macro(FindSalomeMESH)
```

Comment:
Should be `FindSalomeSMESH` (also the file name should be renamed)

Reply:
Good catch, I made this change locally but failed to add it to the git repo.